### PR TITLE
Revert "Delivery_to_lockers"

### PR DESCRIPTION
**Pull request by SashaYan10**
Reverts msenyk/sf-practicum-2024#13

### CODE REVIEW

File: Delivery_to_lockers.app-meta.xml [hunk 1]
```diff
@@ -1,25 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (msenyk) [inline Delivery_to_lockers.app-meta.xml:1]: Why did you delete this file?

Comment (SashaYan10) [reply Delivery_to_lockers.app-meta.xml:1]: I accidentally rushed and merged the pull request without waiting for the review.
